### PR TITLE
modtool_rename: Add list of sensible blocknames

### PR DESCRIPTION
**Pull request by swap-nil7**
Added the script for autocompletion of sensible blocknames for renaming.

### CODE REVIEW

File: gr-utils/python/modtool/modtool_rename.py [hunk 1]
```diff
@@ -38,6 +41,17 @@ def __init__(self):
         self._skip_cmakefiles = False
         self._license_file = None
 
+    def get_block_candidates(self):
+        cpp_blocks = filter(lambda x: not (x.startswith('qa_') or
+                            x.startswith('test_')),
+                            glob.glob1("lib", "*.cc"))
```
Comment (mbr0wn) [inline gr-utils/python/modtool/modtool_rename.py]: Maybe add `*.cpp` to the glob. It's rarely used, and not created by modtool, but it's the more common ending for C++ files and someone might be using it.
Or not. I don't feel strongly about this.

Comment (swap-nil7) [reply gr-utils/python/modtool/modtool_rename.py]: I have updated the same for the inclusion of `cpp` files.

**Comment by mbr0wn** [comment]
Manually rebased. I'm not sure it's working properly, though.

**Comment by mbr0wn** [comment]
(It's not breaking anything, either).

**Pull request closed without merging**